app/styles/layout/l-homepage.scss

```diff
@@ -3,7 +3,9 @@
 
 .infomap {
   align-items: center;
-  background-color: $color-8;
+  background-image: url('../../assets/images/background_the_map.png');
```

Comment:
we are waiting on an image from the client here. we cannot put a picture of our own map........